maven-plugin/src/main/java/com/spotify/missinglink/maven/PackageFilter.java

```diff
@@ -69,12 +69,12 @@ public void setPackage(String name) {
     this.name = name;
   }
 
-  public boolean isIgnoreSubpackages() {
-    return ignoreSubpackages;
+  public boolean isFilterSubpackages() {
+    return filterSubpackages;
   }
 
-  public void setIgnoreSubpackages(boolean ignoreSubpackages) {
-    this.ignoreSubpackages = ignoreSubpackages;
+  public void setFilterSubpackages(boolean filterSubpackages) {
```

Comment:
I wonder how hard it would be to support the old XML property name as well - if I remember right, I think Maven is basically doing some sort of XML-to-java-bean translation so that `<filterSubpackages>true</filterSubpackages>` results in calling this setter method - so perhaps it would be possible to support the old XML property name by just adding an extra setter matching the old name too? i.e.

```java
public void setIgnoreSubpackages(boolean ignoreSubpackages) {
  this.filterSubpackages = ignoreSubpackages;
}
```

I think the actual class renaming is fine, that is probably invisible to Maven more or less

Reply:
that's a cool idea - it seems to work too!